spec/controllers/v2/users/sessions_controller_spec.rb

```diff
@@ -33,5 +33,31 @@
         it { expect(response_hash['error']).not_to be_empty }
       end
     end
+
```

Comment:
Well, since we can sign in with a token as well (i.e. generate a new token), we should test that code path as well IMHO.